packages/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -223,11 +223,26 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
   }
 
   public render(): JSX.Element {
-    const { tickValues, tickFormat, eventAnnotationProps, legendProps, data } = this.props;
+    // eslint-disable-next-line @fluentui/max-len
```

Comment:
This eslint comment should not be needed

Reply:
Will revert this

Reply:
reverted lint override